### DISCUSSION TITLE
01

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#189] [General]
# 潮流周刊第01期 - 安吉黄昏

封面图摄于安吉晚饭黄昏，伴随着一点凉风吹来，还挺有意境的。

https://weekly.liulei.org/posts/01-%E5%AE%89%E5%90%89%E9%BB%84%E6%98%8F

**Comment by vychodlc** [comment]
从200多期看到第1期，真的感觉过了好久好久，从遍地ai到各种各样的前端案例、框架、技术，感觉tw老师的博文记录了时代的变化，太棒了（我的收藏夹已经被老师的分享给塞满了😁）
